climada/entity/exposures/crop_production.py

```diff
@@ -546,14 +645,19 @@ def normalize_with_fao_cp(exp_firr, exp_noirr, input_dir=INPUT_DIR,
         exp_tot_production(list): Exposure crop production value per country
             (before normalization)
     """
-
-    # if the exposure unit is USD/y temporarily reset the exposure to t/y
+    if not input_dir:
+        input_dir = INPUT_DIR
+    if yearrange is None:
+        yearrange = (2008, 2018)
+    if not unit:
+        unit = 't'
+    # if the exposure unit is USD/y or kcal/y, temporarily reset the exposure to t/y
     # (stored in tonnes_per_year) in order to normalize with FAO crop production
-    # values and then apply set_to_USD() for the normalized exposure to restore the
+    # values and then apply set_to_XXX() for the normalized exposure to restore the
     # initial exposure unit
-    if exp_firr.value_unit == 'USD / y':
+    if exp_firr.value_unit == 'USD / y' or 'kcal' in exp_firr.value_unit:
         exp_firr.value = exp_firr.tonnes_per_year
-    if exp_noirr.value_unit == 'USD / y':
+    if exp_noirr.value_unit == 'USD / y' or 'kcal' in exp_noirr.value_unit:
```

Comment:
Why would the value but set to `tonnes_per_year` if the unit is `USD / y` ?

Reply:
we had this discussion in the last code review already, here is the explanatory comment we added as a consequence:

    # if the exposure unit is USD/y or kcal/y, temporarily reset the exposure to t/y
    # (stored in tonnes_per_year) in order to normalize with FAO crop production
    # values and then apply set_to_XXX() for the normalized exposure to restore the
    # initial exposure unit

Reply:
Aaa yes. I am still not very happy with this, but ok.